src/app/services/URL.ts

```diff
@@ -0,0 +1,13 @@
+const APIkey = 'a993fb2adc4cfdf4377bba9ca8e5c1fd';
```

Comment:
Keeping this key inhere is  a **very** bad idea. 
We need to move this to env variables